microsoft-365/enterprise/office-365-network-mac-perf-score.md

```diff
@@ -35,28 +36,26 @@ A very low network assessment value suggests that Microsoft 365 clients will hav
+The **Assessment history** shows the past 30 days of the assessment and the benchmark. You can also report on the metrics history for any office location for up to two years using the history tab. The history tab allows you to select your attributes to report on and by choosing a report timeframe, you can highlight the impact of a network update project and see the improvement to your network assessment.
 
-## Tenant network assessments and office location network assessments
+## Tenant network assessments, office location and remote worker network assessments
 
-A network assessment measures the design of the network perimeter of an office location to Microsoft's network. Improvements to the network perimeter is best done at each office location.
+A network assessment measures the design of the network perimeter of a location to Microsoft's network. Improvements to the network perimeter of office locations is best done at each office location.
 
-We show a network assessment value for the whole Microsoft 365 tenant on the network performance overview page which is a weighted average of the network assessments for all office locations. There is also a specific network assessment value for each detected office location on that location's summary page.
+We show a network assessment value for the whole Microsoft 365 tenant on the network performance overview page which is a weighted average of the network assessments for all office locations. There is also a specific network assessment value for each detected office location on that location's summary page as well as the network assessment value for remote workers in that city. We do not identify individual users working from home.
 
 ## Exchange Online
 
-For Exchange Online the TCP latency from the client machine to the Exchange service front door is measured. This can be impacted by the distance the network travels over the customers LAN and WAN. It can also be impacted by network intermediary devices or services which delay the connectivity or cause packets to be resent. And it is impacted by how far away the nearest Exchange service front door is. The median (also known as the 50th percentile or P50 measure) is taken for all measurements over the previous three days.
+For Exchange Online, the TCP latency from the client machine to the Exchange service front door is measured. This can be impacted by the distance the network travels over the customers LAN and WAN. It can also be impacted by network intermediary devices or services which delay the connectivity or cause packets to be resent. And it is impacted by how far away the nearest Exchange service front door is. The median (also known as the 50th percentile or P50 measure) is taken for all measurements over the previous three days.
```

Comment:
```suggestion
For Exchange Online, the TCP latency from the client machine to the Exchange service front door is measured. This can be impacted by the distance the network travels over the customer's LAN and WAN. It can also be impacted by network intermediary devices or services which delay the connectivity or cause packets to be resent. And it is impacted by how far away the nearest Exchange service front door is. The median (also known as the 50th percentile or P50 measure) is taken for all measurements over the previous three days.
```

Reply:
Johans edit looks good